src/main/java/dwbh/api/domain/input/UpdateGroupInput.java

```diff
@@ -0,0 +1,312 @@
+/*
+ * Copyright (C) 2019 Kaleidos Open Source SL
+ *
+ * This file is part of Don't Worry Be Happy (DWBH).
+ * DWBH is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * DWBH is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with DWBH.  If not, see <https://www.gnu.org/licenses/>
+ */
+package dwbh.api.domain.input;
+
+import java.time.DayOfWeek;
+import java.time.OffsetTime;
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+
+/**
+ * Group input. It contains the fields for a Group
+ *
+ * @since 0.1.0
+ */
+public class UpdateGroupInput {
```

Comment:
Maybe There should be an `UpsertGroupInput` if you would like to reuse most of the properties. See `persistence` comment

Reply:
Changed